src/scripts/slurm_submit.sh

```diff
@@ -85,8 +85,17 @@ bls_add_job_wrapper
 ###############################################################
 
 datenow=`date +%Y%m%d`
-jobID=`${slurm_binpath}/sbatch $bls_tmp_file` # actual submission
-retcode=$?
+retry=0
+MAX_RETRY=3
+until [ $retry -eq $MAX_RETRY ] ; do
```

Comment:
This looks like the number of attempts to submit is 3 but the number of retries is actually 2, right?